<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWindow</class>
 <widget class="QDialog" name="SecondWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Вас приветствует &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; font-style:italic; text-decoration: underline;&quot;&gt;Arch Creator &lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;!&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Эта программа поможет Вам создать чертёж арки для дверного или оконного проёма.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;	Для создания чертежа, введите значения ширини и высоты проёма, а так же стрелы подёма арки, в соответствкющие поля. Значения указываются в миллиметрах, натуральными числани от 1 до 10 000 включительно. Ниже представлен эскиз арки, на котором указаны требуемые параметры.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;img src=&quot;:/new/Арка.png&quot;/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;	После ввода значений, кликните на кнопку Фильтр. Программа выберет арки, которые возможно построить, при заданых параметрах.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;	Обратите внимание на Каталог арок. Фон арок, для которых возможно построение, стал белым. Кликните на нужную Вам арку. Перед Вами появится чертёж выбраной арки. На нём в миллиметрах указаны все нужные, для построения, значения. Часть арок возможно построить исполюзуя дополнительные построения из геометрических фигур, а некоторорые арки задаются указаными уравнениями. Также на чертеже присутствует иформация о соединении криволинейной и прямоугольно частей арки. Чем ближе значение угла к 0, теп плавнее переход.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;	Вы можете сохранить чертёж к себе на компьютер, для этого кликните на кнопку Сохранить и выберите нужную папку.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;	Вы можете создать несколько чертежей одновременно, а так же изменить значение параметров. После каждого изменения, кликайте на кнопку Фильтр.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Спасибо, что выбираете нас !&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Параметры арки</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Стрела подъёма, мм</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="arrow"/>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="text">
           <string>Ширина, мм</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="width"/>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Высота, мм</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="height"/>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Фильтр</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Каталог арок</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_10">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_13">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_12">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_4">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_7">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButton_5">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_6">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_9">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButton_11">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_8">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>165</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
